pkg/api/v1/types.go

```diff
@@ -2013,17 +2023,18 @@ func (spec *KubevirtNodeSpec) MarshalJSON() ([]byte, error) {
 	}
 
 	res := struct {
-		FlavorName                  string             `json:"flavorName"`
-		FlavorProfile               string             `json:"flavorProfile"`
-		CPUs                        string             `json:"cpus"`
-		Memory                      string             `json:"memory"`
-		PrimaryDiskOSImage          string             `json:"primaryDiskOSImage"`
-		PrimaryDiskStorageClassName string             `json:"primaryDiskStorageClassName"`
-		PrimaryDiskSize             string             `json:"primaryDiskSize"`
-		SecondaryDisks              []SecondaryDisks   `json:"secondaryDisks"`
-		PodAffinityPreset           string             `json:"podAffinityPreset"`
-		PodAntiAffinityPreset       string             `json:"podAntiAffinityPreset"`
-		NodeAffinityPreset          NodeAffinityPreset `json:"nodeAffinityPreset"`
+		FlavorName                  string                     `json:"flavorName"`
+		FlavorProfile               string                     `json:"flavorProfile"`
+		CPUs                        string                     `json:"cpus"`
+		Memory                      string                     `json:"memory"`
+		PrimaryDiskOSImage          string                     `json:"primaryDiskOSImage"`
+		PrimaryDiskStorageClassName string                     `json:"primaryDiskStorageClassName"`
+		PrimaryDiskSize             string                     `json:"primaryDiskSize"`
+		SecondaryDisks              []SecondaryDisks           `json:"secondaryDisks"`
+		PodAffinityPreset           string                     `json:"podAffinityPreset"`
```

Comment:
As we prepare the deprecation, why not removing the handling of `PodAffinityPreset` and `PodAntiAffinityPreset` in here too ?
It makes sense to keep the fields in `types.go` to not break the UI until the migration is done, but do see any drawback by removing here already, so it's clear that it's not used?

Reply:
No drawbacks to removing from here. I will remove it.